config/breakerflags_test.go

```diff
@@ -124,6 +172,8 @@ func Test_breakerFlags_Set(t *testing.T) {
 				if cmp.Equal(b[0], tt.want) == false {
 					t.Errorf("breakerFlags.Set() got v, want v, %v", cmp.Diff(b[0], tt.want))
 				}
+			} else if tt.errString != err.Error() {
+				t.Errorf("Failed to get error string: %v", err)
```

Comment:
Maybe it makes sense to print tt.errString as well

Reply:
fixed